LICENSE.txt

```diff
@@ -35,7 +35,7 @@ Mozilla Public License Version 2.0
     means any form of the work other than Source Code Form.
 
 1.7. "Larger Work"
-    means a work that combines Covered Software with other material, in 
+    means a work that combines Covered Software with other material, in
```

Comment:
I'm not going to try to correct the MPL, if that's what you're saying. I just removed the trailing whitespace.